lib/nexmo_markdown_renderer/services/doc_finder.rb

```diff
@@ -92,7 +92,7 @@ def self.configure
 
       def self.load_english
         paths.each do |path|
-          if defined?(Rails) && path.starts_with?("#{Rails.root}/app/views")
+          if defined?(Rails::Application) && path.start_with?("#{Rails.root}/app/views")
```

Comment:
Why do we need to specify `Rails::Application` specifically?

Reply:
because `Rails` is defined, it's a module included by `actionview`